authentication/managing_cloud_provider_credentials/cco-mode-mint.adoc

```diff
@@ -42,7 +42,7 @@ The credential you provide for mint mode in Amazon Web Services (AWS) must have
 * `iam:SimulatePrincipalPolicy`
 ====
 
-The credential you provide for mint mode in Google Cloud Platform (GCP) must have the following permissions:
+The credential you provide for mint mode in Google Cloud Platform (GCP) must have the **IAM Role Administrator** role and the following permissions:
```

Comment:
**IAM Role Administrator** should be added to **Required roles for the installation program**
https://docs.openshift.com/container-platform/4.14/installing/installing_gcp/installing-gcp-account.html#installation-gcp-permissions_installing-gcp-account